BrowserKit/Sources/ToolbarKit/LocationView.swift

```diff
@@ -28,11 +28,14 @@ class LocationView: UIView, UITextFieldDelegate, ThemeApplicable {
         static let horizontalSpace: CGFloat = 16
         static let gradientViewVerticalPadding: CGFloat = 8
         static let gradientViewWidth: CGFloat = 40
+        static let transitionDuration: TimeInterval = 0.3
     }
 
+    private var urlAbsolutePath: String?
     private var notifyTextChanged: (() -> Void)?
     private var locationViewDelegate: LocationViewDelegate?
 
+    private lazy var urlTextFieldSubdomainColor: UIColor = .systemRed
```

Comment:
Is that still needed? The color doesn't match our design system.

Reply:
This is just a default value to avoid the need of unwrapping the variable. The actual value, is set in the `applyTheme` method.

Reply:
Can we maybe use `.clear` instead?